CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to the Project
+
+Hey there! Welcome to the project! Here's some things to keep in mind before you get started.
+
+# Questions
+
+If you have a question, and there isn't any specific Issue that the question is about, please post your questions in the [discussion](https://github.com/orgs/alveusgg/discussions).
```

Comment:
```suggestion
If you're looking to work on an issue but have a question about it or are generally unsure about how to approach it, reach out in that specific issue and folks will be able to help.

For general questions about the project unrelated to a specific issue, please start a [discussion](https://github.com/orgs/alveusgg/discussions) (or file an issue if you believe what you're asking is something actionable in code).
```
Rewording this to push folks towards issues primarily if there is an appropriate one, before falling back to discussions.